src/components/Header.jsx

```diff
@@ -2,19 +2,19 @@ import React from "react";
 import logoImg from "../images/education-logo.png";
 import UserImg from "../images/user.png";
 
+import { HeaderWrapper, ProfileLink, NavListWrapper } from "./style.ts";
```

Comment:
There is no need to include file extension with file path.

Reply:
done